ex25/script.js

```diff
@@ -0,0 +1,30 @@
+function botao(){
+    document.getElementById("EscolhaNumero").style.display = "inline"
+}
+
+function Calcular() {
+    let n = document.getElementById('Numero')
+    let numero = Number(n.value)
+    n.value = ''
+    resultado = document.getElementById('ResultadoNumero')
+    
+    if(verificar(numero)){
+        resultado.innerHTML = 'Seu numero <strong>é primo</strong>'
+    } else {
+        resultado.innerHTML = 'Seu numero <strong>não é primo</strong>'
+    }
+}
+
+function verificar(numero) {
+    if (numero <= 1) {
+        return false
+
+    } else {
+        for (let c = 2; c < numero; c++) {
+            if (numero % c == 0){
+                return false
+            }
+        }
+        return true
+    } 
+}
```

Comment:
- aqui entra naquele caso do return junto ao if. Como se entrar no if, ja dara um return, então nao precisa do else

```suggestion
function verificar(numero) {
    if (numero <= 1) {
        return false
    } 
        
     for (let c = 2; c < numero; c++) {
            if (numero % c == 0){
                return false
            }
        }
        return true
}
```

- outro ponto é que no codigo, a gente sempre tentar dar nomes declarativos onde, so olhando aquilo, eu consiga entender levemente o que faz. Ao olhar o nome da funcao como _verificar_, eu sei que uma verificacao ta sendo feita, mas nao sei exatamente qual. Sera que faria sentido dar um nome mais declarado, ou ate mesmo quebrar a validacao em funcoes menores onde, cada funcao, tenha um objetivo bem definido? vc consegue enxergar possíveis pontos de mudança?